x/evm/types/msg.go

```diff
@@ -37,6 +37,24 @@ func NewTx(
 	gasLimit uint64, gasPrice *big.Int, input []byte, accesses *ethtypes.AccessList,
 ) *MsgEthereumTx {
 	return newMsgEthereumTx(chainID, nonce, to, amount, gasLimit, gasPrice, input, accesses)
+
+}
+
+// UnwrapEthereumMsg extract MsgEthereumTx from wrapping sdk.Tx
+func UnwrapEthereumMsg(tx *sdk.Tx) (*MsgEthereumTx, error) {
```

Comment:
can we add a test for this function? 🙏 and maybe move it to `utils.go`

Reply:
done